packages/core/src/utilities/background-color/background-color.example.html

```diff
@@ -7,12 +7,14 @@
 'secondary-lightest'], ['error', 'error-dark', 'error-darkest', 'error-light', 'error-lighter',
 'error-lightest'], ['base', 'white', 'background', 'background-inverse', 'transparent'] ]; -%> <%
 groups.forEach(group => { -%>
-<section class="ds-l-row ds-u-margin-x--0 ds-u-margin-bottom--3">
+<section class="ds-u-border-bottom--1 ds-u-padding-bottom--3 ds-u-margin-bottom--4">
   <% group.forEach(color => { -%>
-  <article class="ds-l-col--12 ds-l-sm-col--6 ds-u-padding--0 ds-u-margin-bottom--2">
-    <code class="ds-u-margin-bottom--1">.ds-u-fill--<%= color %></code>
-    <span class="ds-u-color--muted js-swatch-hex">#</span>
-    <div class="c-swatch__preview c-swatch__preview--condensed ds-u-fill--<%= color %>"></div>
+  <article class="ds-u-margin-bottom--1">
+    <div
+      class="c-swatch__preview c-swatch__preview--condensed ds-u-radius--circle ds-u-fill--<%= color %>"
+    ></div>
+    <code class="ds-u-fill--transparent">.ds-u-fill--<%= color %></code>
```

Comment:
This reminds me that maybe we should work on adding some documentation on how to create/style examples, but one of the informal style rules I've been using is to try to use `code` styling whenever showing a classname in an example (with a few exceptions like when the classname is being used to show the styling like )

I know it can clash a little sometimes though, especially in the color pages. What do you think about keeping this pattern here for consistency?

Reply:
so are you saying to remove the <code> element from around the HEX value and only use it on classes?

Reply:
Sorry, I mean to remove `ds-u-fill--transparent`